contracts/test/governance/token/erc20/governance-erc20.ts

```diff
@@ -43,15 +43,15 @@ describe('GovernanceERC20', function () {
     signers = await ethers.getSigners();
     GovernanceERC20 = new GovernanceERC20__factory(signers[0]);
 
-    dao = await deployWithProxy<DAO>(new DAO__factory(signers[0]));
-    await dao.initialize([], signers[0].address, addressZero, 'exampleURI');
+    dao = await deployUUPSProxy<DAO>(new DAO__factory(signers[0]), {
+      initializerName: 'initialize',
+      args: [[], signers[0].address, addressZero, 'exampleURI'],
+    });
```

Comment:
This would be the alternative using hardhat upgrades (that we could encapsulate into `deployUUPSProxy`
```suggestion
   dao = (await upgrades.deployProxy(
      new DAO__factory(signers[0]),
      [[], signers[0].address, addressZero, 'exampleURI'],
      {
        kind: 'uups',
        initializer: 'initialize',
        unsafeAllow: ['constructor'],
        constructorArgs: [],
      }
    )) as unknown as DAO;
```

However, hardhat upgrades does not support minimal clones and we are always shown this warning that is not relevant:
```
Warning: Potentially unsafe deployment of @aragon/osx/packages/contracts/src/core/dao/DAO.sol:DAO

    You are using the `unsafeAllow.constructor` flag.
```